javalib/src/main/scala/java/util/StringTokenizer.scala

```diff
@@ -1,103 +1,93 @@
 package java.util
 
-class StringTokenizer(string: String,
-                      private var delimiters: String,
-                      returnDelimiters: Boolean)
-    extends java.util.Enumeration[Object] {
+class StringTokenizer(
+  str: String,
+  private var delim: String,
+  returnDelims: Boolean
+) extends java.util.Enumeration[Object] {
 
-  def this(string: String) = this(string, " \t\n\r\f", false)
-  def this(string: String, delimeters: String) =
-    this(string, delimeters, false)
+  def this(str: String) = this(str, " \t\n\r\f", false)
```

Comment:
It is a matter of style & of the scars, physical and mental, that one bears...
When dealing with spaces/blanks, I usually prefer to use the Unicode  idiom "\u0020",
perhaps with a trailing //  "\u0020" == space.    I find that text editors, formatters, etc. tend to ding spaces, even inside of strings, far too often than my patience will bear.
Of course, decoding Unicode hex on the fly is an acquired taste.